src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -200,8 +204,51 @@ extension ExposureDetectionViewController {
 			checkButton.isEnabled = true
 		case .manual:
 			footerView.isHidden = false
-			checkButton.setTitle(AppStrings.ExposureDetection.buttonRefresh, for: .normal)
-			checkButton.isEnabled = riskProvider.manualExposureDetectionState == .possible
+
+			let nextRefresh = riskProvider.nextExposureDetectionDate()
+			let now = Date()
+
+			if nextRefresh > now, countdown == nil {
```

Comment:
Why must it explicitly be checked that the countdown is nil here?

Reply:
Would prefer having `countdown == nil` linebreak

Reply:
i want to avoid scheduling multiple countdown timers